### ISSUE TITLE
Cant converter all the field for a specific type.

### ISSUE THREAD

**Issue by hatharom** [modelmapper/modelmapper#486]
Ok, its not a bug but a question or feature request.

Is it possible to convert all the occurence of a specific type **without** specificing the locations?

With this we will convert from the specified getter to a setter.
`typeMap.addMappings(mapper->mapper.using(CONVERTER).map(getter, setter));`

The problem I want to solve is like this:

`typeMap.addGlobalMapping(mapper->mapper.using(CONVERTER).map(Source.class, Destination.class));`

So basically without specifing the occurence of the field, all of the source field for a specific type must be converted to  the destination. Matching fields would be simple in my case because source always have the same name as destination.

I have hundred of classes with ton of fields and would be tedious each time explicitly describe the getter,setter mappings.
